src/main/drivers/accgyro_spi_mpu6000.c

```diff
@@ -119,8 +121,25 @@ bool mpu6000ReadRegister(uint8_t reg, uint8_t length, uint8_t *data)
     return true;
 }
 
-void mpu6000SpiGyroInit(uint8_t lpf)
+void mpu6000SpiGyroInit(gyro_t *gyro, uint8_t lpf)
 {
+    uint16_t intPeriod;
+    switch(lpf) {
+        case 0:
+            intPeriod = PERIOD_HZ(8000);
```

Comment:
Yeah, i have to say I think the macro names could be a bit more helpful too.

suggestions?